src/NUnitFramework/framework/Internal/TestParameters.cs

```diff
@@ -123,23 +123,29 @@ private void InitializeArguments(object[] args)
 
         #endregion
 
-        #region IApplyToTest Members
-
         /// <summary>
-        /// Applies ParameterSet values to the test itself.
+        /// Applies the encapsulated parameters to the test method.
```

Comment:
`TestMethod`

Reply:
:+1: This change gets reversed in a later commit.